CONTRIBUTING.md

```diff
@@ -0,0 +1,203 @@
+How to contribute to ezomero
+=================================
+
+Developing Open Source is great fun!  Join us on the [image.sc forums](https://forum.image.sc) and tell us
+which of the following challenges you'd like to solve.
+
+* Mentoring is available for those new to scientific programming in Python.
+* If you're looking for something to implement or to fix, you can browse the
+  [open issues on GitHub](https://github.com/TheJacksonLaboratory/ezomero/issues?q=is%3Aopen).
+* The technical detail of the development process is summed up below.
+
+
+Stylistic Guidelines
+--------------------
+
+* Functions should strive to, whenever possible, have arguments and returns in common Python types, or in numpy/scipy types. No passing OMERO objects back and forth!
+
+* If there is a function from the base [OMERO-py API](https://downloads.openmicroscopy.org/omero/5.6.3/api/python/) that does something, we do not need to duplicate it. Wrapping OMERO-py functions for simpler input/output is fine.
```

Comment:
"If there is a function from the base [OMERO-py API](https://downloads.openmicroscopy.org/omero/5.6.3/api/python/) that does something,"

I might add: "e.g., return an `omero.gateway.ImageWrapper` or similar with `conn.getObject`"